pkg/cmd/build/cmd.go

```diff
@@ -0,0 +1,314 @@
+package build
```

Comment:
Would probably put this under /pkg/build since it's basically a library. Currently all packages under /pkg/cmd have their own main-package and main function which this doesn't.

Reply:
Gotya. Moved :)